instrumentation/pg/lib/opentelemetry/instrumentation/pg/patches/connection.rb

```diff
@@ -111,7 +111,9 @@ def span_attrs(kind, *args)
 
           def extract_operation(sql)
             # From: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/9244a08a8d014afe26b82b91cf86e407c2599d73/plugins/node/opentelemetry-instrumentation-pg/src/utils.ts#L35
-            sql.to_s.split[0].to_s.upcase
+            # Ignores prepend comment
+            comment_regex = %r{\A\/\*.*?\*\/}m
+            sql.to_s.sub(comment_regex, '').split[0].to_s.upcase
```

Comment:
To confirm, we are only interested in removing the first set of long form comments (without newlines) like those generated in SQL Commenter format.

We are also not concerned about inline comments `-- this is a comment`

Is that correct?